content/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning.md

```diff
@@ -180,7 +180,7 @@ jobs:
     runs-on: [self-hosted, ubuntu-latest]
 ```
 
-{% data variables.product.prodname_codeql %} {% data variables.product.prodname_code_scanning %} supports the latest versions of Ubuntu, Windows, and macOS. Typical values for this setting are therefore: `ubuntu-latest`, `windows-latest`, and `macos-latest`. For more information, see {% ifversion ghes %}"[Workflow syntax for GitHub Actions](/actions/reference/workflow-syntax-for-github-actions#self-hosted-runners)" and "[Using labels with self-hosted runners](/actions/hosting-your-own-runners/using-labels-with-self-hosted-runners){% else %}"[Workflow syntax for GitHub Actions](/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on){% endif %}."
+{% data variables.product.prodname_codeql %} {% data variables.product.prodname_code_scanning %} supports the latest versions of Ubuntu, Windows, and macOS. Typical values for this setting are therefore: `ubuntu-latest`, `windows-latest`, and `macos-latest`. For more information, see {% ifversion ghes %}"[Workflow syntax for GitHub Actions](/actions/reference/workflow-syntax-for-github-actions#self-hosted-runners)" and "[Using labels with self-hosted runners](/actions/hosting-your-own-runners/using-labels-with-self-hosted-runners){% else %}"[Choosing the runner for a job](/actions/using-jobs/choosing-the-runner-for-a-job){% endif %}."
```

Comment:
Not sure whether this is correct, but the conditions covered by the `{% else %}` here are FPT (which we want), GHEC and GHAE. Do we want to link to "Choosing the runner for a job" for GHEC AND GHAE too?

I'll defer to the author of the issue here as I'm not sure what the best article we should be linking to for GHAE.

Reply:
@cmwilson21  and @mchammer01 - great questions. Apologies for not considering the versioning when I raised the issue. As far as I can tell, both the articles [Choosing the runner for a job](/actions/using-jobs/choosing-the-runner-for-a-job) and [Using labels with self-hosted runners](/actions/hosting-your-own-runners/using-labels-with-self-hosted-runners) are now available for all versions of the docs. So it looks as if we could simplify this and remove the conditions entirely, so that we link to both articles for all versions.

I _think_ that when this was first written maybe we didn't support self-hosted runners on GitHub.com or GHAE, but they are clearly supported now.

Reply:
@mchammer01 and @felicitymay - thank you both for weighing in on this! I removed the versioning, which in turn did remove the reference for "[Workflow syntax for GitHub Actions](/actions/reference/workflow-syntax-for-github-actions#self-hosted-runners)". Let me know if you still want that included in the links as well.

I also repositioned the "For more information" line in the very next paragraph. It was on it's own line, but I removed the break as I thought it made sense for it to be included with the paragraph it was referring to, the self-hosted runners bit. If that needs to be reverted, let me know and I'm happy to change it back. I'm adding screenshots to show the difference for your reference. 

Thank you both, again!

![Screen Shot 2022-04-13 at 1 33 22 PM](https://user-images.githubusercontent.com/77312589/163248597-69505895-aee9-48e5-821e-5d1ceed4b4ac.png)
![Screen Shot 2022-04-13 at 1 39 13 PM](https://user-images.githubusercontent.com/77312589/163248626-1a686b66-2a1f-4dc7-8851-0c1ff55c67db.png)